src/components/AssetSearch/AssetRow.tsx

```diff
@@ -1,31 +1,25 @@
 import { Box, Button, Text, useColorModeValue } from '@chakra-ui/react'
-import { fromAssetId } from '@shapeshiftoss/caip'
 import { Asset } from '@shapeshiftoss/types'
-import { useRouteMatch } from 'react-router-dom'
+import { useParams } from 'react-router-dom'
 import { ListChildComponentProps } from 'react-window'
 import { AssetIcon } from 'components/AssetIcon'
+import { MatchParams } from 'pages/Accounts/Account'
 
 export const AssetRow: React.FC<ListChildComponentProps> = ({ data, index, style }) => {
-  const match = useRouteMatch<{ address: string; network: string }>()
+  const { assetId: searchedAssetId } = useParams<MatchParams>()
```

Comment:
This currently evaluates to undefined, tried merging latest develop in and got the same.